src/main/resources/swagger/thurloe.yaml

```diff
@@ -115,17 +112,20 @@ paths:
             "required": ["key", "value"],
             "properties": {
               "userId": {
-                "type": "string"
+                type: "string"
               },
-              "keyValuePair": {
-                "type": "object",
-                "required": [ "key", "value" ],
-                "properties": {
-                  "key": {
-                    "type": "string"
-                  },
-                  "value": {
-                    "type": "string"
+              "keyValuePairs": {
```

Comment:
This YAML file is pretty inconsistent even with itself, so let's clean up when we touch. No quotes around keys, remove EOL commas.